### PR TITLE
Edit home

### PR DESCRIPTION
**Pull request by Richa-rani02**
Added Edit notes functionality
responsive home page

### CODE REVIEW

**Review by tanishk15sharma — APPROVED** [review]
code is clear and well describing .
I liked your naming conventions

**Review by KanekarSnehal — APPROVED** [review]
Code is well structured
Good use of context.
Code looks good to merge

File: src/pages/Notes.jsx [hunk 1]
```diff
@@ -41,19 +42,45 @@ const {notesState:{notesList},setNotes,noteInput}=useNotes();
...
+                        <button type="button" class="text-white bg-cyan-700 hover:bg-cyan-800 focus:ring-4 focus:outline-none focus:ring-blue-300 font-medium rounded-lg text-sm px-5 py-2.5 text-center inline-flex items-center mr-2 dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800 relative" onClick={(e)=>{
+                            tooglenotesModal(e);
+                            setNotes(noteInput)
+                            }}>
+                           Add Notes
+                            <IoMdAdd size={22} className="ml-2"/>
+                        </button>
+                    </div>
+                    
+                    <div class="flex pr-6">
```
Comment (sharathnair9999) [inline src/pages/Notes.jsx]: I see few class instead of className. You can change that

**Review by sharathnair9999 — APPROVED** [review]
Everything looks good

File: src/pages/Notes.jsx (hunk 1 above)
Comment (Richa-rani02) [reply src/pages/Notes.jsx]: fixed .Thanks for pointing out.

**Pull request merged**